tests/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ else()
     set(libname "config")
 endif()
 
+include_directories(${CMAKE_SOURCE_DIR}/lib)
```

Comment:
The [examples](https://github.com/hyperrealm/libconfig/blob/master/examples/c/CMakeLists.txt) seem to set the include directory per-target:

```cmake
target_include_directories(libconfig_tests PRIVATE ${CMAKE_SOURCE_DIR}/lib)
```